src/models/network.js

```diff
@@ -43,6 +44,7 @@ function Network(attr) {
 		},
 		chanCache: [],
 		ignoreList: [],
+		keepNick: null,
```

Comment:
Make sense to default it to false rather than null?

Reply:
Using null means its `?string` instead of `string|bool`, which I think makes more sense here.

Reply:
Oh sorry, I had thought this was a config to let them decide to have keepNick or not, ignore me. Good point.